reapy/core/project/marker.py

```diff
@@ -7,18 +7,18 @@ class Marker(ReapyObject):
 
     _class_name = "Marker"
 
-    def __init__(
-        self, parent_project=None, index=None, parent_project_id=None
-    ):
+    def __init__(self, parent_project=None, index=None,
+                 parent_project_id=None):
         if parent_project_id is None:
             message = (
                 "One of `parent_project` or `parent_project_id` must be "
-                "specified."
-            )
+                "specified.")
```

Comment:
I don't think this is either nice, recommended, or commonly used... I would leave the parenthesis alone as before.